extensions/machine-learning-services/src/configurations/config.ts

```diff
@@ -30,7 +30,7 @@ export class Config {
 	 * Loads the config values
 	 */
 	public async load(): Promise<void> {
-		const rawConfig = await fs.readFile(path.join(this._root, 'src', 'configurations', configFileName));
+		const rawConfig = await fs.readFile(path.join(this._root, 'out', 'configurations', configFileName));
```

Comment:
Normally we put these at the root so you don't have to hardcode the paths like this

Reply:
yeah I put it inside the configuration folder. but will fix later in other PRs

---
In reply to: [369282200](https://github.com/microsoft/azuredatastudio/pull/8920#discussion_r369282200) [](ancestors = 369282200)